Foundation/NSURLSession/http/EasyHandle.swift

```diff
@@ -90,8 +87,8 @@ extension _EasyHandle {
 }
 
 internal extension _EasyHandle {
-    func completedTransfer(withErrorCode errorCode: Int?) {
-        delegate?.transferCompleted(withErrorCode: errorCode)
+    func completedTransfer(withErrorCode errorCode: Int?, errorDesc: String?) {
```

Comment:
Here and elsewhere, this should be spelled out: `errorDescription`, as it is in the rest of the project.